app/routes/blog.$slug.tsx

```diff
@@ -7,19 +8,57 @@ import invariant from "tiny-invariant";
 import { PUBLISH_DATE_FORMATTER } from "~/utils";
 import type { Mdx } from "blog/parser";
 
+export const shortCacheMaxAge = `max-age=28800`; // 8 hour cache
+export const longCacheMaxAge = `max-age=604800`; // One week cache
```

Comment:
naming nitpick: i'd go with `const EIGHT_HOUR_CACHE_AGE` and `const ONE_WEEK_CACHE_AGE` to be explicit.

Reply:
This change was made.